packages/vuetify/src/components/VBtn/VBtn.tsx

```diff
@@ -113,6 +113,17 @@ export const VBtn = defineComponent({
       return props.variant === 'elevated' && !(props.disabled || props.flat || props.border)
     })
 
+    const buttonId = String(Math.random()).replace('.', '')
+
```

Comment:
this could be extracted into a separate function `generateRandomId` but I wanted to check first where would be the correct place for this helper.

Reply:
What is the purpose of a random ID?

Reply:
> What is the purpose of a random ID?

The docs state: 

>The value used when the component is selected in a group. If not provided, a unique ID will be used.

Reply:
It already uses the index of the item in the group.